### DISCUSSION TITLE
References in footnotes -- [Article Title]{.nocase} formatting issue?

### DISCUSSION THREAD

**Discussion by NixImagery** [squidfunk/mkdocs-material#4103] [Q&A] [answered]
Hi, I'm using mkdocs-material to build a static academic site and it works beautifully. One small niggle is the cited references produced in the footer of a page have what looks like unintended formatting information.

Example expected output:
1. Feynman R P 1985 _"Surely You're Joking, Mr. Feynman!": Adventures of a Curious Character_ (New York: W. W. Norton) ::: 

Actual output:
1. Feynman R P 1985 _[\"Surely You're Joking, Mr. Feynman!\": Adventures of a Curious Character]{.nocase}_ (New York: W. W. Norton) ::: 

Best I can do to show you what's happening. There are backlinks at the end of the line, and they work fine. It's the wrapping of the article title in square brackets with what looks like a markdown-style class modifier _.nocase_ that I'm not able to get rid of. It appears like that shown above, no matter what csl file I am using.

Is this a bug/feature of pybtex?

Any advice welcome. Thanks in anticipation.

**Comment by squidfunk** [comment, marked as answer]
That sounds weird. We're not doing anything like that. I'd recommend asking the maintainers of `pybtex`.

**Reply by NixImagery** [reply]
The solution is to upgrade `mkdocs-bibtex`. See [shyamd/mkdocs-bibtex issue 14](https://github.com/shyamd/mkdocs-bibtex/issues/144) for details.
